internal/pkg/model/manifest.go

```diff
@@ -18,8 +18,8 @@ type RecordPaths interface {
 	GetParentPath() string // parent path relative to the project dir
 }
 
-// Record - manifest record.
-type Record interface {
+// ObjectManifest - manifest record for a object.
+type ObjectManifest interface {
```

Comment:
Fixol som nekonzistentny naming, ... `Record` reprezentuje zaznam z `manifest.json` pre konkretny objekt (config, row ...), preto som to premenoval na `ObjectManifest`.